Xamarin.Forms.Platform.UAP/ListViewRenderer.cs

```diff
@@ -168,11 +184,6 @@ protected override void OnElementPropertyChanged(object sender, PropertyChangedE
 			{
 				ClearSizeEstimate();
 			}
-			else if (e.PropertyName == ListView.ItemsSourceProperty.PropertyName)
-			{
-				ClearSizeEstimate();
```

Comment:
what happens if I set the ItemsSource to a new Collection?

Reply:
@rmarinho I had the same concern above, but see @PureWeen's reply to my comment.
I wast just concerned that if ItemsSource was set to a new collection, the ListView would not be reloaded.

Reply:
https://github.com/xamarin/Xamarin.Forms/pull/3023#discussion_r195190863

If you set a new ItemSource *TemplatedItemList* propagates a reset to *ListViewRenderer.OnCollectionChanged* which calls *ReloadData*

Having the *ReloadData* call here was causing it to be called twice which was leading to this issue

https://github.com/xamarin/Xamarin.Forms/issues/3009

Reply:
Thanks @PureWeen.
Just confirming the case of an initial null ItemsSource, then set to new collection is covered?

Reply:
It does but I didn't have a test for null initially just being set to null then back so I added one just to be thorough

https://github.com/xamarin/Xamarin.Forms/pull/3023/files#diff-0edd99bdcd5cbb81e6610e90df74169aR149

Thanks for the added test case!!